README.md

```diff
@@ -1,31 +1,32 @@
-
 <img src="public_html/assets/img/logo/nf-core-logo.png" width="400">
 
 # [nf-co.re](https://github.com/nf-core/nf-co.re)
 
 This repository contains code for the nf-core website: **http://nf-co.re/**
 
 ## Packages used
+
 Here's how the website is built:
 
-* Language: PHP
-* HTML / CSS / JS framework: [Bootstrap v4](http://getbootstrap.com/)
-* JavaScript libraries:
-    * [jQuery](https://jquery.com/)
-    * [Popper.js](https://popper.js.org/) _(used for bootstrap tooltips)_
-    * [highlightjs](https://highlightjs.org/) _(syntax highlighting)_
-    * [Leaflet](https://leafletjs.com/) _(contributor map)_
-    * [Moment.js](https://momentjs.com/) _(time and date parsing)_
-    * [Chart.js](https://www.chartjs.org/) _(statistics plots)_
-    * [hammer.js](https://hammerjs.github.io/) _(mobile touch interaction handling)_
-    * [chartjs-plugin-zoom](https://github.com/chartjs/chartjs-plugin-zoom) _(Zoom and pan plugin for Chart.js)_
-    * [Canvas2Svg.js](https://gliffy.github.io/canvas2svg/) _(SVG exports of Chart.JS plots)_
-    * [FileSaver.js](https://github.com/eligrey/FileSaver.js/) _(Trigger browser downloads from in-page data, used to save plot SVGs to files)_
-    * [jQuery table sorter](https://mottie.github.io/tablesorter/) _(sorting tables)_
-* PHP Markdown parsing: [Parsedown](https://github.com/erusev/parsedown/) and [Parsedown Extra](https://github.com/erusev/parsedown-extra/)
-* SVG icons: http://www.flaticon.com, https://worldvectorlogo.com/
+- Language: PHP
+- HTML / CSS / JS framework: [Bootstrap v4](http://getbootstrap.com/)
+- JavaScript libraries:
+  - [jQuery](https://jquery.com/)
+  - [Popper.js](https://popper.js.org/) _(used for bootstrap tooltips)_
+  - [highlightjs](https://highlightjs.org/) _(syntax highlighting)_
+  - [Leaflet](https://leafletjs.com/) _(contributor map)_
+  - [Moment.js](https://momentjs.com/) _(time and date parsing)_
+  - [Chart.js](https://www.chartjs.org/) _(statistics plots)_
+  - [hammer.js](https://hammerjs.github.io/) _(mobile touch interaction handling)_
+  - [chartjs-plugin-zoom](https://github.com/chartjs/chartjs-plugin-zoom) _(Zoom and pan plugin for Chart.js)_
+  - [Canvas2Svg.js](https://gliffy.github.io/canvas2svg/) _(SVG exports of Chart.JS plots)_
+  - [FileSaver.js](https://github.com/eligrey/FileSaver.js/) _(Trigger browser downloads from in-page data, used to save plot SVGs to files)_
+  - [jQuery table sorter](https://mottie.github.io/tablesorter/) _(sorting tables)_
+- PHP Markdown parsing: [Parsedown](https://github.com/erusev/parsedown/) and [Parsedown Extra](https://github.com/erusev/parsedown-extra/)
+- SVG icons: http://www.flaticon.com, https://worldvectorlogo.com/
```

Comment:
strangely enough, this got tweaked by `VsCode` `format-on-save` facility. 🤔

Reply:
Ah yes, `*` and `-` are equivalent in markdown for lists. Although it makes no difference to the output, we typically use `*` basically everywhere in @nf-core repos, so I would prefer it if you could revert these changes please :+1:

The extra whitespace after headings etc. is good (now checked by our markdown linter on some repos), so that can stay. 

Phil

Reply:
Sure, no problem 👍